lib/file-helper.js

```diff
@@ -107,16 +110,12 @@ const getComponentData = async (componentName) => {
   }
 
   // Read from disk
-  try {
-    const yamlPath = join(configPaths.components, componentName, `${componentName}.yaml`)
-    const yamlData = yaml.load(await readFile(yamlPath, 'utf8'), { json: true })
+  const yamlPath = join(configPaths.components, componentName, `${componentName}.yaml`)
+  const yamlData = yaml.load(await readFile(yamlPath, 'utf8'), { json: true })
 
-    return {
-      name: componentName,
-      ...yamlData
-    }
-  } catch (error) {
-    throw new Error(error)
```

Comment:
Removed the try/catch as it wraps and re-throws the error that would have thrown anyway